src/featbin/extract-feature-segments.cc

```diff
@@ -45,6 +45,8 @@ int main(int argc, char *argv[]) {
     BaseFloat min_segment_length = 0.1,  // Minimum segment length in seconds.
         max_overshoot = 0.0;  // max time by which last segment can overshoot
     BaseFloat samp_freq = 100;  // feature sampling frequency (assuming 10ms window shift)
```

Comment:
Thanks, I didn't know about the --snip-edges option in compute-mfcc-feats. This actually explains why the extract-feature-segments program seems to have worked before without the need for such option.